docs/topics/class-based-views/intro.txt

```diff
@@ -93,7 +93,8 @@ or raises :class:`~django.http.HttpResponseNotAllowed` if not::
 
     # urls.py
     from django.urls import path
-    from myapp.views import MyView
+
+    from .views import MyView
```

Comment:
Leave alone. This looks like a root URLConf.

Reply:
ok